pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: pagseguro_smart_flutter
 description: Integre o Flutter com maquinas smarts da Pagseguro
-version: 1.0.0
+version: 1.0.2
```

Comment:
I think is better to use 1.1.0, because pix is a new feature.

Reply:
Ok